v4-client-py/examples/composite_example.py

```diff
@@ -29,25 +29,26 @@ async def main() -> None:
     subaccount = Subaccount(wallet, 0)
     ordersParams = loadJson('human_readable_orders.json')
```

Comment:
Basically reverted this file (`composite_example_.py`) to what it was before @lucas-dydx  modified it with the short term example. There are two place_order calls though, let me know if one should be deleted.

Reply:
IIRC you can keep them both since it was originally like that